lib/lookup/src/lookup_v2/owned.rs

```diff
@@ -238,6 +282,11 @@ mod test {
             (r#"foo."a\"a"."b\\b".bar"#, r#"foo."a\"a"."b\\b".bar"#),
             ("<invalid>", "<invalid>"),
             (r#""🤖""#, r#""🤖""#),
+            (".(a|b)", "(a|b)"),
+            (".(a|b|c)", "(a|b|c)"),
+            ("foo.(a|b|c)", "foo.(a|b|c)"),
+            ("[0].(a|b|c)", "[0].(a|b|c)"),
+            (".(a|b|c).foo", "(a|b|c).foo"),
```

Comment:
Probably a good idea to add a few tests with whitespacing, to ensure that is parsed as expected.

Reply:
These are the serialization tests. There is a whitespace test on the parser already. I'll add another here though.